c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -30,9 +30,11 @@ namespace CUDACachingAllocator {
 //   split. If no block is found, the allocator will delegate to cudaMalloc.
 // - If the cudaMalloc fails, the allocator will free all cached blocks that
 //   are not split and retry the allocation.
-// - Large (>1MB) and small allocation requests are handled separately. Large
-//   allocation requests can be filled by a cudaMalloc call of the exact size.
-//   Small requests will allocate and split a 1MB buffer, if necessary.
+// - Large (>1MB) and small allocations are stored in separate free lists.
```

Comment:
Should "free lists" be "pools" to be consistent with the new naming?
Same thing for the comment above the `free_block` and `cacheInfoAux` functions.